src/ws/cockpitwebservice.c

```diff
@@ -581,6 +581,14 @@ process_transport_authorize (CockpitWebService *self,
       g_bytes_unref (payload);
     }
 
+  /* only give the password once; if the superuser bridge asks again, the password is either wrong, or PAM has
```

Comment:
I *think* this logic checks out.  It flies a little bit against the whole logic of the superuser-init-complete thing, though... I wonder if @mvollmer would have something to say about this.

Reply:
The init-complete thing would be for the case where sudo doesn't even ask once., no? We still want to wipe the password as soon as possible in that case.